samples/subsys/usb/webusb/README.rst

```diff
@@ -37,15 +37,15 @@ Build and flash webusb sample with:
    :goals: flash
    :compact:
 
-Testing with latest Google Chrome on host
-*****************************************
+Testing with the latest Google Chrome on host
+*********************************************
 
-This sample application requires latest Google Chrome, a web page
+This sample application requires the latest Google Chrome, a web page
```

Comment:
would it make sense to reword along the lines of ".... a modern browser, supporting WebUSB <link to https://caniuse.com/webusb>" ? Since Edge is also supported, it might make sense to be slightly more broad here.

Reply:
Is this https://caniuse.com/ credible enough to be put in the docs?

Reply:
ya good point. I think it's trustworthy but we shouldn't necessarily link to a 3rd party unless really needed. It's fine to stick to "latest Chrome" wording, I guess.